frontend/packages/component-library/src/spacer/stories/Spacer.story.tsx

```diff
@@ -0,0 +1,60 @@
+import type {Meta, StoryObj} from '@storybook/react';
+import {within, expect} from '@storybook/test';
+
+import Spacer, {SpacerProps} from '..';
```

Comment:
nit: for consistency would be nice to update it to
```suggestion
import Spacer, {SpacerProps} from './../Spacer';
```